packages/headless/src/features/facets/facet-search-set/generic/generic-facet-search-validate-payload.ts

```diff
@@ -5,5 +5,6 @@ export const facetSearchOptionsDefinition = {
   facetId: facetIdDefinition,
   captions: new RecordValue({options: {required: false}}),
   numberOfValues: new NumberValue({required: false, min: 1}),
+  initialNumberOfValues: new NumberValue({required: false, min: 1}),
```

Comment:
If `initialNumberOfValues` is controlled inside of the reducer, it does not need to be part of the definition. I don't think the value can be set from the outside?